backend/backend/common/room.py

```diff
@@ -28,17 +34,45 @@ def room_id(self):
     def add_member(self, member: RoomMember):
         self._members.update({member.id: member})
 
-    def update_leader(self, player_id, new_leader_id):
+    def update_leader(self, player_id: str, new_leader_id: str):
         player = self._members.get(player_id)
         new_leader = self._members.get(new_leader_id)
 
         if not player or not new_leader:
-            raise PlayerNotInRoom(players=[player_id, new_leader_id])
+            raise PlayerNotInRoomError(players=[player_id, new_leader_id])
 
         if not player.leader:
-            raise PlayerIsNotLeader(player_name=player.name)
+            raise PlayerIsNotLeaderError(player_name=player.name)
 
         player.leader = False
         new_leader.leader = True
 
+    def get_room_member(self, player_id: str) -> RoomMember:
+        member = self._members.get(player_id)
+        if not member:
+            raise PlayerNotInRoomError(players=[player_id])
+
+        return member
+
+    async def _broadcast(self, message: BaseModel):
+        message = Message(timestamp=datetime.now(),
```

Comment:
I think that `Message` default timestamp should be now, there is no need to restate it everytime we are creating message, unless datetime was already given to us